__tests__/tests/reducers/notifications.reducer.js

```diff
@@ -0,0 +1,475 @@
+import React from 'react';
```

Comment:
Generally speaking, I agree with @andrewda about not nesting describes, that can get quite messy pretty quickly.
I think one describe per test suite is fine and just using `it` clauses should be enough for describing a good test suite. Can you please flatten your tests so you only use 1 describe and multiple `it` statements?

Reply:
Sure